packages/react-renderer-demo/firebaseFunctions.js

```diff
@@ -1,96 +0,0 @@
-const functions = require('firebase-functions');
-const admin = require('firebase-admin');
-const next = require('next');
-const { join } = require('path');
-const { parse } = require('url');
-
-const isDev = process.env.NODE_ENV !== 'production';
-// cant use next.config.mjs because its a ES module
-const distDir = '../dist';
-const nextjsDistDir = join('src', distDir);
-const nextjsServer = next({
-  dev: isDev,
-  conf: {
-    distDir: nextjsDistDir,
-  },
-});
-const nextjsHandle = nextjsServer.getRequestHandler();
-
-admin.initializeApp();
-
-const db = admin.firestore();
-
-exports.nextjsFunc = functions.https.onRequest((req, res) => {
-  console.log('File: ' + req.originalUrl); // eslint-disable-line no-console
-  return nextjsServer.prepare().then(() => {
-    const parsedUrl = parse(req.url, true);
-    const { pathname } = parsedUrl;
-
-    if (pathname === '/notifications') {
-      const endDate = Number(req.query.end);
-      return db
-        .collection('notifications')
-        .doc('notifications')
-        .get()
-        .then((snapshot) => {
-          const data = snapshot
-            .data()
-            .notifications.filter(({ active_till: activeTill }) => {
-              const dueDate = activeTill.toDate();
-              return dueDate.getTime() >= endDate;
-            })
-            .map(({ 'created-at': createdAt, active_till: activeTill, ...notification }) => ({
-              ...notification,
-              activeTill: activeTill.toDate(),
-              'created-at': createdAt ? createdAt.toDate : undefined, // eslint-disable-line camelcase
-            }));
-          res.status(200).json(data);
-          res.finished = true;
-          return res.end();
-        });
-    }
-
-    // handle GET request to /service-worker.js
-    if (pathname === '/service-worker.js') {
-      const filePath = join(__dirname, '.next', pathname);
-
-      nextjsServer.serveStatic(req, res, filePath);
-    } else {
-      nextjsHandle(req, res, parsedUrl);
-    }
-  });
-});
-
-const github = require('@actions/github');
-
-exports.sendComment = functions.https.onRequest(async (request, response) => {
```

Comment:
@Hyperkid123 We should keep this part (and migrate it to somewhere else) as it handles the bot.

Is it possible to migrate it to vercel as well? 🤔

Reply:
https://vercel.com/docs/concepts/functions/serverless-functions

Reply:
I can't do it now I am on meetings whole week. If you create an issue I can work on it later. I just want the build to stop failing.

Reply:
Yeah, actually we can remove it - as long we won't remove anything from the firebase itself, it should be fine.